spec/behaviour_raker_rspec.rb

```diff
@@ -0,0 +1,140 @@
+require 'spec_helper'
```

Comment:
Vito, I was having problem with path. I have not lose problem with RVM or your link. I have used 'sudo bundle update' and 'sudo bundle exec rspec'. Yep, I know it is not correct, but it allowed to avoid problem with pathes. I've checked with rails capybara works normal on my PC withiut 'sudo'.